src/main/java/com/azure/cosmos/examples/modeling/async/Deployment.java

```diff
@@ -0,0 +1,310 @@
+        DatabaseSchema_3.add(new SchemaDetails("leases", "/id"));
+        DatabaseSchema_3.add(new SchemaDetails("customer", "/id"));
+        DatabaseSchema_3.add(new SchemaDetails("product", "/categoryId"));
+        DatabaseSchema_3.add(new SchemaDetails("productCategory", "/type"));
+        DatabaseSchema_3.add(new SchemaDetails("productTag", "/type"));
+        DatabaseSchema_3.add(new SchemaDetails("salesOrder", "/customerId"));
+
+        List<SchemaDetails> DatabaseSchema_4 = new ArrayList<>();
+        DatabaseSchema_4.add(new SchemaDetails("customer", "/customerId"));
+        DatabaseSchema_4.add(new SchemaDetails("product", "/categoryId"));
+        DatabaseSchema_4.add(new SchemaDetails("productMeta", "/type"));
+        DatabaseSchema_4.add(new SchemaDetails("salesByCategory", "/categoryId"));
+
+        DatabaseSchema.add(DatabaseSchema_1);
+        DatabaseSchema.add(DatabaseSchema_2);
+        DatabaseSchema.add(DatabaseSchema_3);
+        DatabaseSchema.add(DatabaseSchema_4);
+
+        return DatabaseSchema;
+
+    }
+
+    public void CreateDatabaseAndContainers(CosmosAsyncClient cosmosDBClient, String databaseName, int schema) {
+
+        System.out.println("creating database and containers for schema v" + schema);
+        System.out.println("DatabaseName:" + database + " key:provided");
+
+        List<List<SchemaDetails>> DatabaseSchema = getSchemaDetails();
+        if (schema >= 1 & schema <= 4) {
+            ThroughputProperties throughputProperties = ThroughputProperties.createAutoscaledThroughput(4000);
+
+            Mono<CosmosDatabaseResponse> databaseIfNotExists = cosmosDBClient.createDatabaseIfNotExists(databaseName);
+            databaseIfNotExists.flatMap(databaseResponse -> {
+                database = cosmosDBClient.getDatabase(databaseResponse.getProperties().getId());
+                logger.info("Checking database " + database.getId() + " completed!\n");
+                return Mono.empty();
+            }).block();
+            
+            for (SchemaDetails schemaDetails : DatabaseSchema.get(schema - 1)) {
+                CosmosContainerProperties autoScaleContainerProperties = new CosmosContainerProperties(
+                        schemaDetails.getContainerName(), schemaDetails.getPk());
+                Mono<CosmosContainerResponse> containerIfNotExists = database.createContainerIfNotExists(autoScaleContainerProperties, throughputProperties);
+
+                //  Create autoscale container with 4000 RU/s
+                CosmosContainerResponse cosmosContainerResponse = containerIfNotExists.block();
+                CosmosAsyncContainer container = database.getContainer(cosmosContainerResponse.getProperties().getId());
+                System.out.println("container: " + cosmosContainerResponse.getProperties().getId() + "." + container.getId() + " created!");
+            }
+        }
+    }
+
+    public void DeleteDatabases(CosmosAsyncClient cosmosDBClient, int schemaVersion) {
```

Comment:
nit: I think the the method name should start with lowercase for java convention `DeleteDatabases` -> `deleteDatabases `

Reply:
True - has been done very inconsistent (we missed this during initial review). syn Deployment has some methods named like in C# some like in Java. For any new public surface area I would change it now - not sure whether it is justified to touch existing surface area thought.

Reply:
Ok thanks, will change. I ported this all from C#, missed the Java convention.